app/controllers/applications_controller.rb

```diff
@@ -9,7 +9,8 @@ def show
       @searched_pets = []
     end
     @applicant = Applicant.retrieve_applicant(params[:id])
-    @pets = list_pets(@applicant)
+    # @pets = list_pets(@applicant)
+    @pets = @applicant.pets
```

Comment:
man, can't believe it is this simple with Active Record haha. I have a hard time remembering to use these easy solutions.